docs/src/tutorials/tut_setup.md

```diff
@@ -1,13 +1,320 @@
 # Setting up your simulation from scratch
 
-## Hydrostatic tank
+In this tutorial, we will guide you through the general structure of simulation files.
+We will set up a simulation similar to the example simulation
+[`examples/fluid/hydrostatic_water_column_2d.jl`](https://github.com/trixi-framework/TrixiParticles.jl/blob/main/examples/fluid/hydrostatic_water_column_2d.jl),
+which is one of our simplest example simulations.
+In the second part of this tutorial, we will show how to replace components
+of TrixiParticles.jl by custom implementations from within a simulation file,
+without ever cloning the repository.
 
+For different setups and physics, have a look at [our other example files](@ref examples).
 
+## Resolution
 
+At the beginning of most simulation files, we define the numerical resolution,
+so that it can easily be found and changed.
+First, we import TrixiParticles.jl and
+[OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl), which we will
+use at the very end for the time integration.
+```jldoctest tut_setup; output = false
+using TrixiParticles
+using OrdinaryDiffEq
 
-# Example file
-```julia
-!!include:examples/fluid/hydrostatic_water_column_2d.jl!!
+# output
 
 ```
+Now, we define the particle spacing, which is our numerical resolution.
+We also set the number of boundary layers, which need to be sufficiently
+large, depending on the smoothing kernel and smoothing length, so that
+the compact support of the smoothing kernel is fully sampled with particles
+for a fluid particle close to a boundary.
```

Comment:
This would be much more concrete if you would just write compact_support < boundary layers and give here the common values for the compact support or how one can find them.